src/gists/gist.jl

```diff
@@ -0,0 +1,79 @@
+type Gist <: GitHubType
+    url::Nullable{HttpCommon.URI}
+    forks_url::Nullable{HttpCommon.URI}
+    commits_url::Nullable{HttpCommon.URI}
+    id::Nullable{GitHubString}
+    description::Nullable{GitHubString}
+    public::Nullable{Bool}
+    owner::Nullable{Owner}
+    user::Nullable{Owner}
+    truncated::Nullable{Bool}
+    comments::Nullable{Int}
+    comments_url::Nullable{HttpCommon.URI}
+    html_url::Nullable{HttpCommon.URI}
+    git_pull_url::Nullable{HttpCommon.URI}
+    git_push_url::Nullable{HttpCommon.URI}
+    created_at::Nullable{Dates.DateTime}
+    updated_at::Nullable{Dates.DateTime}
+    forks::Nullable{Vector{Gist}}
+    files::Nullable{Dict}
+    history::Nullable{Vector{Dict}}
```

Comment:
One of the keys in this Dict is `"user"`. This could be mapped to a GitHubType`Owner`. Should I do that in the `Gist` constructor?